### PR TITLE
Add NekoWiki namespace, #17

### PR DESCRIPTION
**Pull request by Awkan**
As mentionned in issue #17 , I updated the application namespace.    
    
@Nek- One question remains to me: Shall I change the services name ? From `app.xxxxxxx` to  `nekland.neko_wiki.xxxx` for example ?

### CODE REVIEW

**Comment by Nek-** [comment]
@Awkan I think it's a really good idea to change services names. With whole namespace or just "neko_wiki"?

File: composer.json [hunk 1]
```diff
@@ -4,7 +4,9 @@
     "type": "project",
     "description": "A simple wiki written in Symfony",
     "autoload": {
-        "psr-0": { "": "src/" }
+        "psr-4": {
```
Comment (Nek-) [inline composer.json]: top

**Comment by Awkan** [comment]
@Nek- Maybe `neko_wiki.xxxx` would be shorter, I'll modify it !

**Comment by Awkan** [comment]
@Nek- Done, you can have a look ;)

**Comment by Nek-** [comment]
You're awesome. 😄

**Pull request merged**